iron-collapse.html

```diff
@@ -128,6 +128,14 @@
       return this.horizontal ? 'width' : 'height';
     },
 
+    get dimensionMax() {
```

Comment:
I will make `dimensionMax` and `dimensionMaxCss` private, but leave them as properties. They are needed in more than one of the functions. It would be a DRY violation, otherwise.

`dimension` was part of the public api before and still is. Changing the return values from width/heigth to maxWidth/maxHeight would be a breaking change. So I favor not to go that path.